### DISCUSSION TITLE
Installing InvenTree using Docker. Running into error "no such service: inventree-server"

### DISCUSSION THREAD

**Discussion by miracuruzrh** [inventree/InvenTree#7067] [Q&A] [answered]
Hello

I'm following the Docker installation guide to install InvenTree.
At the step [Initial Database Setup](https://docs.inventree.org/en/latest/start/docker_install/#initial-database-setup) I'm running into the error: "no such service: inventree-server", when executing the command `docker compose run --rm inventree-server invoke update`. 

Installed is
Docker version 26.0.2, build 3c863ff
Docker Compose version v2.26.1

If anyone has an idea, what I do wrong, would be much apprechiated to get a little help.
Please let me know, if further information is needed.

Best regards Simon

**Comment by SchrodingersGat** [comment]
Are you definitely running the command from within the same directory that the `docker-compose` file is located?

**Reply by miracuruzrh** [reply]
Sorry that I not have yet written. I was able to solve it. I thought, the .env file needs in my local InvenTree folder also to be in the subfolder Contrib/container like in the Github repository. Then I realized, that I need to move the file into the root directory of my docker compose folder. Then it worked. I'm still in the installation process. I wonder for the Caddy file. I guess it needs also to be in the same folder?

**Reply by SchrodingersGat** [reply, marked as answer]
Yes all three files need to be located in the same directory

**Reply by miracuruzrh** [reply]
Thank you.

**Comment by anjanpa** [comment]
error: wrong number of arguments, should be 2

I am getting above error after running this command:

1. docker compose -f .devcontainer/docker-compose.yml up -d 
2. docker logs devcontainer-inventree-1

I am  using the latest version of the inventree.

**Reply by matmair** [reply]
1) far to less info
2) please open a new discussion, this is a year old and seemingly unrelated
